lib/beaker-pe/install/pe_utils.rb

```diff
@@ -289,7 +289,11 @@ def fetch_pe(hosts, opts)
         # @param [Host] host The host to install pacakges for
         # @api private
         def deploy_frictionless_to_master(host)
-          klass = host['platform'].gsub(/-/, '_').gsub(/\./,'')
+          platform = host['platform']
+          if platform == "aix-7.2-power"
```

Comment:
Can you leave a comment with this code change to state this 7.2 needs uses 7.1 classification? I also believe that you had mentioned plans to improve the AIX installers to not care about the AIX version, which could also result in this cod eventually being removed, you know, in the *future*. 👾